packages/react-google-maps-api/src/useLoadScript.ts

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react'
+
+import { injectScript } from './utils/injectscript'
+import { preventGoogleFonts } from './utils/prevent-google-fonts'
+
+import { isBrowser } from './utils/isbrowser'
+import { LoadScriptProps, defaultLoadScriptProps } from './LoadScript'
+
+export const useLoadScript = ({
+  id = defaultLoadScriptProps.id,
+  version = defaultLoadScriptProps.version,
+  googleMapsApiKey,
+  language,
+  region,
+  libraries,
+  preventGoogleFontsLoading,
+  onError,
+  onLoad,
+  onUnmount
+}: LoadScriptProps) => {
+  const isMounted = React.useRef(false)
+  const [isLoaded, setLoaded] = React.useState(false)
+
+  React.useEffect(() => {
+    isMounted.current = true
+    return () => {
+      isMounted.current = false
+      if (onUnmount) {
+        onUnmount()
+      }
+    }
+  }, [])
+
+  React.useEffect(() => {
+    if (isBrowser && preventGoogleFontsLoading) {
```

Comment:
I haven't fully understood why is this needed, so I kept it, but `react-google-maps` had nothing similar so I wonder what's the purpose of it?

Reply:
That is my own addition, cos I do not want to google maps to include google font Roboto. I already have Roboto fonts loaded with async in my app.